### PR TITLE
Friend request

### PR DESCRIPTION
**Pull request by Blodispic**
- Friend request now use the information in friend request
- When clicking on accept or refuse, make 
- Now add friends in the friends relations

### CODE REVIEW

File: back/src/Oauth/Oauth.service.ts [hunk 1]
```diff
@@ -76,6 +76,7 @@ export class OauthService {
       email: data.email,
       intra_avatar: data.image.link,
       access_token: token
+
```
Comment (emmanuel-lbs) [inline back/src/Oauth/Oauth.service.ts:79]: cool cool

File: front/src/components/connection/Sign.tsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ export default function Sign() {
 
 
     const fetch_name_avatar = async (e: any) => {
+
```
Comment (emmanuel-lbs) [inline front/src/components/connection/Sign.tsx:20]: cool cool

**Review by emmanuel-lbs — APPROVED** [review]

**Review by se-lee — APPROVED** [review]
Solid work :clap: :100:  
looks good to me!
Maybe erase our hard-coded Friends in NYC (Ross, Rachel, Joey... everyone) in comments if we don't need them anymore
Nice work! Thanks :tada:

**Review by Acusanno — APPROVED** [review]
It looks good

**Pull request merged**